### ISSUE TITLE
Is openbazaar still alive?

### ISSUE THREAD

**Issue by kallisti5** [OpenBazaar/openbazaar-go#2109]
- Circa 2020 it was quoted as shutting down.
- Circa 2021 no notice of shutdown is on website
- Circa 2021 https://gethaven.app/ no longer functions

_Abandoning open source projects with no clear message, so hot right now_

* If it's dead and the original authors have no interest in maintaining or reviewing patches, mark the git repo as read-only and communicate that via the website.
* If the project is alive, communicate that and  the intent of the developers.

The options above will encourage a fork if the original authors are no longer interested in the project.

tldr; don't leave a project in an "abandoned limbo" because nobody knows what the hell to do.

**Comment by hoffmabc** [comment]
It's dead

**Comment by lonnietc** [comment]
Hello,  

Hope that your day is going well.

I am now working with Openbazaar since I really think that it would make a great base to build from for the project that I am working on and think that it could really be a viable option. To that end, I intend to fork off everything and try to build a very new and competitive option.

For the new approach, a number of changes will be made but the core seems to be perfect for the job and I would like to discuss it more with you.

For now, I just want to bring up some nodes (locally and remotely) to test and have been able to compile the Golang Windows version as well as to start up the node (Master branch) and Windows client (v0.14.10 branch).

Today, I wanted to test the existing openbazaar-go version that is in the Docker Hub (https://hub.docker.com/r/openbazaar/server/)

Did not see any docker run string so I tried:

 docker run --env "USERNAME=toto" --env "PASSWORD=tata" -p 9005:9005 -p 4001:4001 -p 4002:4002 -d openbazaar/server

which seems to bring it up, but could not connect from a Windows client to the VPS that I was testing on.

Also, I am still trying to work out how to build the "openbazaar-desktop" on a development Ubuntu 20.04 box that I have set up.

If you are interested in working together on this then I think that it could be a viable project, but if not then maybe you could give me a bit of guidance on some of these things since I need to get a working core base for running and compiling before moving to the modifications stage for the next steps.

Looking forward to hearing back from anyone interested and in particular with Brian (@hoffmabc) as one of the original founders to see if we might be able to do something together or discuss some ideas together.

Best Regards

**Comment by fcl1892** [comment]
our fork is here. https://github.com/mobazha/multiwallet

**Comment by MichaIng** [comment]
Correcting link to actual Go server repo 😉: https://github.com/mobazha/openbazaar-go
Many thanks for developing the fork ❤️!

Do you aim to make this more popular, promote it within the OpenBazaar community (while probably difficult to reach now 😢), and how would you rate its current state and stability?

I'm asking since we provide(d) an OpenBazaar installation with our distribution [DietPi](https://github.com/MichaIng/DietPi). and I recognised now that the website is down and the project dead. In case the fork gets a notable amount of users, also to have more reviews, bug reports and all such, I'd consider to switch to your fork instead of removing the software option from our project. Removing it would be sad, since OpenBazaar is quite unique 🙂.
